Content.Server/Power/Generator/PortableGeneratorSystem.cs

```diff
@@ -164,6 +166,12 @@ private void GeneratorTugged(EntityUid uid, PortableGeneratorComponent component
         }
     }
 
+    private void OnSwitchPowerCheck(EntityUid uid, FuelGeneratorComponent comp, ref SwitchPowerCheckEvent args)
+    {
+        if (comp.On)
+            args.Message = Loc.GetString("fuel-generator-verb-disable-on");
+    }
```

Comment:
i think its not really intuitive enough that setting `args.Message` will disable the verb, i'd rename the field to `DisableMessage` or something?